packages/venia-concept/src/components/ProductImageCarousel/thumbnail.css

```diff
@@ -1,24 +1,38 @@
+/** The "thumbnai" is a round pil button below the product image in mobile or
```

Comment:
```suggestion
/** The "thumbnail" is a round pil button below the product image in mobile or
```

Reply:
Funny -- I can't commit this suggestion cause it's on Alexey's fork.